pkg/controller/controller.go

```diff
@@ -0,0 +1,175 @@
+package controller
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"sync/atomic"
+
+	"github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
+	certificatesv1beta1 "k8s.io/api/certificates/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	certv1beta1 "k8s.io/client-go/kubernetes/typed/certificates/v1beta1"
```

Comment:
I think this is normally imported as:
```suggestion
	certificatesv1 "k8s.io/client-go/kubernetes/typed/certificates/v1beta1"
```